### ISSUE TITLE
Support for io.js?

### ISSUE THREAD

**Issue by codingtwinky** [FredrikNoren/ungit#490]
[begin religious battle]

So as you may or may not be aware of, node has been forked to io.js. (or is it [f*&@ed](http://wesleyio.tumblr.com/post/104637877991/node-js-is-forked-not-f-ed)?)  And [v1.0 of io.js](https://iojs.org/) is out.  

This shouldn't really impact ungit as ungit doesn't do a lot of heavy server side work and as long as  express works.  But here are things to consider.

* io.js will be [compatible](https://iojs.org/faq.html) with node.js.  So if works in node.js it should work in io.js
* io.js has [iopm](https://www.npmjs.com/package/iopm) but I think it's just alias to npm 
* expressjs, ungit's main serverside component, is now belong to joyent due to recent drama.  So I doubt expressjs will be actively working with io.js.  And i doubt [hapi.js](http://hueniverse.com/2014/12/04/before-the-drama/) will either.

so to sum it up, I don't think there is anything that needs to be done...  But incase I missed something or if anybody want to add more I figure this issue will be a place to be.

[/end religious battle]

**Comment by Ajedi32** [comment]
If io.js is compatible with node, then this shouldn't affect ungit at all, right? Users can install io.js instead of node if they want, then just `npm install  -g ungit` as usual.

**Comment by Ajedi32** [comment]
I do have one question though.

> * expressjs, ungit's main serverside component, is now belong to joyent due to recent drama. So I doubt expressjs will be actively working with io.js. And i doubt [hapi.js](http://hueniverse.com/2014/12/04/before-the-drama/) will either.

You say expressjs belongs to joyent now? Where did you hear that? I can't seem to find any information on that change. (Though from a developer's perspective, I don't think it really affects anything.)

**Comment by codingtwinky** [comment]
Sorry, I guess it moved to [Strongloop](https://github.com/strongloop/express) not Joyent when [previous express maintainer](https://medium.com/code-adventures/farewell-node-js-4ba9e7f3e52b) departed as a maintainer of express.

And yes I don't think there is anything needs to be done so I closed the issue myself for now.  Now that may change if iojs begins to require different `package.json` structure or having it's own package manager rather then using npm alias.
